examples/api/lib/widgets/scroll_view/grid_view.0.dart

```diff
@@ -0,0 +1,190 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math' as math;
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+
+void main() => runApp(const GridViewExampleApp());
+
+class GridViewExampleApp extends StatelessWidget {
+  const GridViewExampleApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      home: Padding(
+        padding: const EdgeInsets.all(20.0),
+        child: Card(
+          elevation: 8.0,
+          child: GridView.builder(
+            padding: const EdgeInsets.all(12.0),
+            gridDelegate: CustomGridDelegate(dimension: 240.0),
+            // Try uncommenting some of these properties to see the effect on the grid:
+            // itemCount: 20, // When not set, the number of grid tiles is infinite.
+            // scrollDirection: Axis.horizontal, // The default is vertical.
+            // reverse: true, // The default is false, going down (or left to right).
```

Comment:
```suggestion
            // itemCount: 20, // When not set, the number of grid tiles is infinite.
            // scrollDirection: Axis.horizontal, // The default is vertical.
            // reverse: true, // The default is false, going down (or left to right).
```

            // itemCount: 20, // When not set, the number of grid tiles is infinite.


` // When not set,`  and `, // The default is false` are a bit unusual in the comment.